apps/frontend/src/pages/landing/landing-page.tsx

```diff
@@ -0,0 +1,16 @@
+import {
+	LandingFooter,
+	LandingHeader,
+	LandingHero,
+} from "./components/components.js";
+import styles from "./styles.module.css";
+
+const LandingPage: React.FC = () => (
+	<main className={styles["landing-page"]} role="main">
+		<LandingHeader />
+		<LandingHero />
+		<LandingFooter />
+	</main>
```

Comment:
Now you have <header> and <footer> inside <main>, which is not correct. <main> should only have the main content (LandingHero in your case)